cli/src/cli.rs

```diff
@@ -8,6 +8,9 @@ use {
     std::io::{Result, Write},
 };
 
+use std::fs::File;
+use std::io::Read;
```

Comment:
To avoid code style conflict between contributors, we gather all `use` into a single line.
Then, `cargo fmt` will do everything for us.

e.g.
```rust
use {                                                          
    crate::{command::Command, helper::CliHelper, print::Print},
    gluesql_core::{                                            
        prelude::Glue,                                         
        store::{GStore, GStoreMut},                            
    },                                                         
    rustyline::{error::ReadlineError, Editor},                 
    std::{                                                     
        fs::File,                                              
        io::{Read, Result, Write},                             
    },                                                         
};
```